### PR TITLE
Live radio e2e metadata ati tests

### LINKED ISSUE #3324: Add cypress test for live radio analytics
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jroebu14**
Resolves https://github.com/bbc/simorgh/issues/3324

**Overall change:** Enables the live radio page metadata and ATI e2e tests.

**Code changes:**

Tests include

- should include correct metadata (checks `description`, `og:title`, `og:type`, `lang`)
- should have the correct facebook metadata
- should have the correct open graph metadata
- should include mainEntityOfPage in the LinkedData
- should have a noscript tag with an 1px image with the ati url
- Made `checkFacebookMetadata` cypress command not specific to article pages.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

**Testing notes**
No testing required.

### CODE REVIEW

File: cypress/integration/pages/liveRadio.js [hunk 1]
```diff
@@ -39,82 +39,70 @@ const runCommonTests = ({ service }) =>
...
           '1609039196070050',
           `${appConfig[service].articleAuthor}`,
         );
       });
 
-      it.skip('should have the correct open graph metadata', () => {
+      it('should have the correct open graph metadata', () => {
         cy.checkOpenGraphMetadata(
-          'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
+          null,
```
Comment (pjlee11) [inline cypress/integration/pages/liveRadio.js:78]: Should this not match the description as per `body.promo.summary,`?

File: cypress/integration/pages/liveRadio.js [hunk 2]
```diff
@@ -39,82 +39,70 @@ const runCommonTests = ({ service }) =>
...
           `${appConfig[service].defaultImage}`,
           `${appConfig[service].defaultImageAltText}`,
           `${appConfig[service].locale}`,
           `${appConfig[service].defaultImageAltText}`,
-          "Meghan's bouquet laid on tomb of unknown warrior",
+          null,
```
Comment (pjlee11) [inline cypress/integration/pages/liveRadio.js:83]: Should this not match the title as per `body.promo.name,`?

File: cypress/integration/pages/liveRadio.js [hunk 3]
```diff
@@ -39,82 +39,70 @@ const runCommonTests = ({ service }) =>
...
         );
       });
 
-      it.skip('should have the correct twitter metadata', () => {
+      it('should have the correct twitter metadata', () => {
         cy.checkTwitterMetadata(
           'summary_large_image',
           `${appConfig[service].twitterCreator}`,
-          'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
+          null,
```
Comment (pjlee11) [inline cypress/integration/pages/liveRadio.js:93]: Similar to above is this not the title value?

File: cypress/integration/pages/liveRadio.js [hunk 4]
```diff
@@ -39,82 +39,70 @@ const runCommonTests = ({ service }) =>
...
           `${appConfig[service].defaultImageAltText}`,
           `${appConfig[service].defaultImage}`,
           `${appConfig[service].twitterSite}`,
-          "Meghan's bouquet laid on tomb of unknown warrior",
+          null,
```
Comment (pjlee11) [inline cypress/integration/pages/liveRadio.js:97]: Is this the description value as above?

**Comment by jamesbhobbs** [comment]
Blocked by https://github.com/bbc/simorgh/pull/3333

**Comment by jamesbhobbs** [comment]
Moved to WIP as there are significant conflicts

**Comment by HarveyPeachey** [comment]
Hey just a heads up that in #3339 I've added e2e tests for ATI for all pages including live radio. Was there a reason why the ATI cypress tests skipped live radio pages?

**Comment by jroebu14** [comment]
Closed in favour of https://github.com/bbc/simorgh/pull/3464

**Pull request closed without merging**